source/test/TestApp/PlayFabEventTest.h

```diff
@@ -24,6 +24,13 @@ namespace PlayFabUnit
 {
     struct TestContext;
 
+    // A wrapper around TestEvents, so we can track it back to the test that launched the event
+    class TestEvent : public PlayFab::PlayFabEvent
+    {
+    public:
+        TestContext* testContext;
```

Comment:
Raw pointer bad!  But I can see how it has come to this.  If you could at least initialize it to {nullptr} it will look less terrifying.

Reply:
A future iteration might be able to use weak_ptr<> or something, but that's too big of a change to make right now.